README.md

```diff
@@ -49,8 +49,11 @@ Basically:
 - 4 Kubernetes minor releases for the management cluster (N - N-3)
 - 6 Kubernetes minor releases for the workload cluster (N - N-5)
 
-**NOTE:** As the versioning for this project is tied to the versioning of Cluster API, future modifications to this policy may be made to more closely align with other providers in the Cluster API
-ecosystem.
+**NOTES:**
+* As the versioning for this project is tied to the versioning of Cluster API, future modifications to this
+  policy may be made to more closely align with other providers in the Cluster API ecosystem.
```

Comment:
Not blocking, but I will drop this sentence (It doesn't seem to add much value to me)

Reply:
Makes sense. I wasn't entirely sure if I should just drop it :D